index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+
+        <nav class="tool_bar">
+            <a href="#" class="my-logo">Egbe Bless Ashuakie</a>
```

Comment:
Only the hamburger button should be in the `nav` tag. Consider moving the logo outside it.